source/include/platform/acmacosx.h

```diff
@@ -119,6 +119,7 @@
 #include "aclinux.h"
 
 #ifdef __APPLE__
+#include <semaphore.h>
 #define sem_destroy         sem_close
```

Comment:
This was initially what I did, simply include semaphore before acpi.h with an #ifdef.  But I thought this might be considered 'hacky' or otherwise might be met with disapproval (for no good reason, obviously.  Sorry, I think I just over thought this entire thing!).  But if you have no problems with that solution, then I'll update my pull request to do this instead.  Thanks!